README.md

```diff
@@ -1 +1,200 @@
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+ 
+- **[Add html webpage]**
+- **[Add readme.md files]**
+- **[Add MIT license file]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+ 
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+**A web Browser
+A text editor
+Basic knowledge of HTML and CSS
+Git (optional)**
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  **git clone https://github.com/peteronyam/Hello-Microverse-Project.git**
+
+
+### Install
+
+Install this project with:
+
+  **npm install**
+
+### Usage
+
+To run the project, execute the following command:
+
+**npm start**
+
+### Run tests
+
+To run tests, run the following command:
+
+  **npm test**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@peteronyam](https://github.com/peteronyam)
+- Twitter: [@jiyapete](https://twitter.com/jiyapete)
+- LinkedIn: [peteronyam](https://linkedin.com/in/peteronyam)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Updating design]**
+- [ ] **[updating styling]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse fir this privilege 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+ 
+
+- **[License file is not working]**
+
+  - [Try to update the link to the license file and also update the license file in your code]
+
+- **[why is CSS styling not responding]**
+
+  - [make sure your CSS styling is having the same classes as used in the html file]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [OPTIONAL] Kindly consider removing this as it is just there to give you on how to set up the MIT license for your repo.